promgen/management/commands/alerts-index.py

```diff
@@ -7,7 +7,21 @@
 
 
 class Command(BaseCommand):
-    def handle(self, **kargs):
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--dry-run',
+            dest='dryrun',
+            action='store_true',
+            help='show what would have been inserted',
+        )
```

Comment:
```suggestion
        parser.add_argument(
            '--force',
            dest='dryrun',
            action='store_false',
            help='Defaults to dry run. Use to execute operation',
        )
```

In `alerts-prune` we use `--force` for our flag, so maybe it's best to match the same here.